src/wp-includes/class-wp-post-type.php

```diff
@@ -669,6 +669,9 @@ public function add_supports() {
 					add_post_type_support( $this->name, $args );
 				}
 			}
+			if ( post_type_supports( $this->name, 'editor' ) && post_type_supports( $this->name, 'custom-fields' ) && post_type_supports( $this->name, 'revisions' ) && ! post_type_supports( $this->name, 'footnotes' ) ) {
```

Comment:
For multiple conditions i personally follow.

```suggestion
			if (
				post_type_supports( $this->name, 'editor' ) &&
				post_type_supports( $this->name, 'custom-fields' ) &&
				post_type_supports( $this->name, 'revisions' ) &&
				! post_type_supports( $this->name, 'footnotes' )
			) {
```

Reply:
Thank you for the suggestion 👍  I'm applying it at https://github.com/WordPress/gutenberg/pull/58882.